plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -410,6 +412,14 @@ public HiveSessionProperties(
                         LEGACY_HIVE_VIEW_TRANSLATION,
                         "Use legacy Hive view translation mechanism",
                         hiveConfig.isLegacyHiveViewTranslation(),
+                        false),
+                booleanProperty(REDIRECT_ICEBERG_TABLES,
+                        "Enable redirecting iceberg tables to a different catalog",
+                        hiveConfig.isRedirectIcebergTables(),
+                        false),
+                stringProperty(TARGET_CATALOG_FOR_ICEBERG_TABLES,
+                        "The target catalog for redirecting iceberg tables",
+                        hiveConfig.getTargetCatalogForIcebergTables(),
```

Comment:
what are security and usability implication of allow users to set this at runtime?
it would be safer to have this as a config property only